### PR TITLE
Feature/issue 14

### PR DESCRIPTION
**Pull request by Edouard-6**

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            </rule>
+          </section>
+        </rules>
+      </arrangement>
+    </codeStyleSettings>
+    <codeStyleSettings language="kotlin">
+      <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
+    </codeStyleSettings>
+  </code_scheme>
+</component>
```
Comment (tepoorte) [inline .idea/codeStyles/Project.xml:123]: add void line at the end

File: .idea/codeStyles/codeStyleConfig.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <state>
+    <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+  </state>
+</component>
```
Comment (tepoorte) [inline .idea/codeStyles/codeStyleConfig.xml:5]: add void line at the end

File: app/src/main/java/com/studhub/app/GreetActivity.kt [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+@Composable
+fun GreetContent(name: String?) {
+    Text(
+        modifier = Modifier.testTag("greetingMessage"),
+        text = "Hello my dear $name, how are you?",
+        color = MaterialTheme.colorScheme.primary
+    )
+}
```
Comment (tepoorte) [inline app/src/main/java/com/studhub/app/GreetActivity.kt:46]: 1)use a different name than the one used on the bootcamp
2) add void line at the end

File: app/src/main/java/com/studhub/app/ui/theme/Type.kt [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        fontWeight = FontWeight.W500,
+        fontSize = 14.sp
+    ),
+    caption = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Normal,
+        fontSize = 12.sp
+    )
+    */
+)
```
Comment (tepoorte) [inline app/src/main/java/com/studhub/app/ui/theme/Type.kt:28]: 1)remove comments
2) add new void line at the end

**Review by tepoorte — COMMENTED** [review]
make sure all build and run correctly. Add void lines when "-" is shown.

File: app/src/main/java/com/studhub/app/GreetActivity.kt (hunk 3 above)
Comment (NervzZ) [inline app/src/main/java/com/studhub/app/GreetActivity.kt:46]: newline to be added

File: app/src/main/java/com/studhub/app/MainActivity.kt [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            }
+            composable("Cart") {
+                AboutScreen()
+            }
+            composable("Map") {
+                MapsActivity()
+            }
+        }
+    }
+}
```
Comment (NervzZ) [inline app/src/main/java/com/studhub/app/MainActivity.kt:61]: newline to be added

File: app/src/main/java/com/studhub/app/ui/AboutScreen.kt [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    StudHubTheme {
+        Column(Modifier.fillMaxSize().padding(16.dp)) {
+            Text(text = "About Us:")
+            Spacer(Modifier.height(16.dp))
+            Text(text = "We are a MarketPlace company working for EPFL campus.")
+            Spacer(Modifier.height(16.dp))
+            Text(text = "Contact us at ...")
+        }
+    }
+}
```
Comment (NervzZ) [inline app/src/main/java/com/studhub/app/ui/AboutScreen.kt:21]: newline to be added

File: app/src/main/java/com/studhub/app/ui/CartScreen.kt [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+@Composable
+fun CartScreen() {
+    StudHubTheme {
+        Column(Modifier.fillMaxSize().padding(16.dp)) {
+            Text(text = "Your Cart:")
+            Spacer(Modifier.height(16.dp))
+            // TODO: Add cart items list
+        }
+    }
+}
```
Comment (NervzZ) [inline app/src/main/java/com/studhub/app/ui/CartScreen.kt:19]: newline to be added

File: app/src/main/java/com/studhub/app/ui/theme/Color.kt [hunk 8]
```diff
@@ -0,0 +1,8 @@
+package com.studhub.app.ui.theme
+
+import androidx.compose.ui.graphics.Color
+
+val Purple200 = Color(0xFFBB86FC)
+val Purple500 = Color(0xFF6200EE)
+val Purple700 = Color(0xFF3700B3)
+val Teal200 = Color(0xFF03DAC5)
```
Comment (NervzZ) [inline app/src/main/java/com/studhub/app/ui/theme/Color.kt:8]: newline to be added

File: app/src/main/java/com/studhub/app/ui/theme/Shape.kt [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Shapes
+import androidx.compose.ui.unit.dp
+
+val Shapes = Shapes(
+    small = RoundedCornerShape(4.dp),
+    medium = RoundedCornerShape(4.dp),
+    large = RoundedCornerShape(0.dp)
+)
```
Comment (NervzZ) [inline app/src/main/java/com/studhub/app/ui/theme/Shape.kt:11]: newline to be added

File: app/src/main/java/com/studhub/app/ui/theme/Theme.kt [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+        LightColorPalette
+    }
+
+    MaterialTheme(
+        colors = colors,
+        typography = Typography,
+        shapes = Shapes,
+        content = content
+    )
+}
```
Comment (NervzZ) [inline app/src/main/java/com/studhub/app/ui/theme/Theme.kt:44]: Are we already changing the theme with that PR? It should be Ray's task to figure that out.

File: app/src/main/java/com/studhub/app/ui/theme/Type.kt (hunk 4 above)
Comment (NervzZ) [inline app/src/main/java/com/studhub/app/ui/theme/Type.kt:28]: newline to be added

**Review by NervzZ — CHANGES_REQUESTED** [review]
Also remove app/src/main/res/layout/activity_main.xml that should not be used anymore

**Review by NervzZ — APPROVED** [review]
LGTM now

**Pull request merged**